R/calculate_annual_pd_changes.R

```diff
@@ -106,5 +120,39 @@ calculate_pd_change_annual <- function(data,
       PD_change = .data$PD_late_sudden - .data$PD_baseline
     )
 
+  exposure_at_default <- exposure_at_default %>%
+    # distinct in order to remove unused technology level information, the
+    # trivial year and the unneeded financial info
+    dplyr::distinct(
+      .data$investor_name, .data$portfolio_name, .data$company_name,
+      .data$ald_sector, .data$scenario_geography, .data$plan_sec_carsten
+    )
+
+  #TODO: needs a check which lines have been removed
+  results <- results %>%
```

Comment:
no new TODOS in PRs please
Also can u please elaborate on when we run risk of losing rows ehre?